tests/quantizer_impl_test.py

```diff
@@ -191,39 +191,39 @@ def test_GetScale_PerChannelScale():
   # Rank1 tensors
   x_r1 = tf.ones([4])
```

Comment:
Since quantized_linear is trying to incorporate binary quantizer as well, I suggest add more scale test here for per channel scales.

1) instead of having input all as 1, consider use wider range of input values so we can test better how scale is calculated; 
2) Add test on quantizer scale during training and eval; to make sure scales don't change between two stages.

Reply:
This is not a test for quantized_linear. I only made a change here because I changed the name of the `_get_scale` function. 

However, these are still good points, and I will add tests with specific scale values to the rest of the quantized_linear tests in the`qactivations_test.py` file

Reply:
Added both (1) and (2)!